### ISSUE TITLE
Can't connect to the postgresql use localhost in docker inside wsl2 

### ISSUE THREAD

**Issue by nulIptr** [dbeaver/dbeaver#8930]
#### System information: 
- Operating system (distribution) and version  win10 2004 
- DBeaver version   DBeaver	7.1.0.202005311732
- Additional extensions 

#### Connection specification:
- Database name and version 
PostgreSQL 12.3 (Debian 12.3-1.pgdg100+1)
PostgreSQL 12.3 (Debian 12.3-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
- Driver name  
PostgreSQL JDBC Driver 42.2.5
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
no

#### Describe the problem you're observing:
I install Ubuntu 20.04 use wsl2 and install docker inside wsl
then run postgresql image and config port like this :  `-p 5432:5432`
DBeaver can't connect to the db use localhost
but in my app it works(I use `Microsoft.EntityFrameworkCore` and `Npgsql.EntityFrameworkCore.PostgreSQL` connect to db)
However ,when I use the local IP of wsl2 like`172.22.57.15`,it works,but, you know,when I restart the WSL2 instance  the local IP will changes
#### Include any warning/errors/backtraces from the logs

<img width="459" src="https://user-images.githubusercontent.com/7925831/84142758-f3741780-aa87-11ea-9071-c54c39e8b7d4.png">
<img width="662" src="https://user-images.githubusercontent.com/7925831/84143230-ca07bb80-aa88-11ea-835b-ab38d52e73d5.png">
I think the key is `localhost`  not equals to `127.0.0.1` when WSL2 enabled

**Comment by serge-rider** [comment]
Similar to #8203

**Comment by ololomp** [comment]
same =(

**Comment by mtowersap** [comment]
I didn't quite have this problem, but I couldn't get dbeaver to connect to the running postgresql container.  realized I had a windows postgresql server running, and it was trying to connect to that instead. I turned it off, and poof, could connect to the container.  I could have gotten around that probably changing the port, but I wasn't using the windows postgres server., so I made it manual and left it off
